spec/requests/bulk_actions/validate_cocina_descriptive_jobs_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bulk validate cocina descriptive' do
+  let(:rendered) do
+    Capybara::Node::Simple.new(response.body)
+  end
+
+  describe 'GET #new' do
+    before do
+      sign_in build(:user), groups: ['sdr:administrator-role']
+    end
+
+    it 'draws the form' do
+      get '/bulk_actions/validate_cocina_descriptive_job/new'
+
+      expect(rendered).to have_css 'textarea[name="description"]'
+      expect(rendered).to have_css 'input[type="file"][name="csv_file"]'
+    end
+  end
+
+  describe 'create' do
+    let(:user) { build(:user) }
+
+    before do
+      sign_in user, groups: ['sdr:administrator-role']
+    end
+
+    it 'creates a job' do
+      params = { 'csv_file' => fixture_file_upload('bulk_upload_descriptive.csv', 'text/csv') }
+
+      expect do
+        post '/bulk_actions/validate_cocina_descriptive_job', params:
+      end.to have_enqueued_job(ValidateCocinaDescriptiveJob)
+        .with(Integer, { csv_file: String })
+      expect(response).to have_http_status(:see_other)
+    end
+
+    context 'when dupolicate columns in csv' do
```

Comment:
Typo: `dupolicate` (though maybe it's a word in Italian I haven't learned yet 🤌)